http.c

```diff
@@ -4226,6 +4241,14 @@ evhttp_get_request_connection(
 	bufferevent_disable(evcon->bufev, EV_WRITE);
 	bufferevent_setfd(evcon->bufev, fd);
 
+	if (0 < http->connection_max_ttl) {
```

Comment:
Let's keep it normal, like everywhere, i.e. `foo > 0` not `0 < foo`

Reply:
That's funny.  Everyone has their own normal.  I never use ``>`` only ``<``.  This is less confusing for me because I always know the smaller thing goes on the left.